examples_wasp/README.md

```diff
@@ -0,0 +1,45 @@
+## Running multi-key load test with Seth and WASP
+To effectively simulate transaction workloads from multiple keys, you can utilize a "rotating wallet." Refer to the [example](client_wasp_test.go) code provided for guidance.
+
+There are 2 modes: Ephemeral and a static keyfile mode
+
+### Ephemeral mode
+We generate 60 ephemeral keys and run the test, set `ephemeral_addresses_number` in `seth.toml`
```

Comment:
maybe add info that this mode should never be used on live networks unless we want to lose funds when the test ends? :-)